kenyan-recipes/ugali.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ugali</title>
+
+    <link rel="stylesheet" href="../css/styles.css"/>
+</head>
+<body>
+    <div><a href="../index.html">Home</a></div>
+    <H1 class="heading-text">Ugali Recipe</H1>
+
+    <h3>Description</h3>
+
+    <p>Ugali is a very common lunch/dinner dish in most of our homes. Cooking it is easy and 
+        can be served with vegetables, meat, soup or even fish</p>
+
+    <img src="../images/ugali.jpeg" alt="matoke meal" width="" height="">
+    
+    <img src="../images/people eating.jpeg" alt="people eating ugali" width="" height="">
+
```

Comment:
Use external Css styling for image attribute

Reply:
- [ ] Always make sure to include an explanation, a "why", for each requested change, and use polite words like `Kindly`, and `It would be better`. Kindly Implement it. 👍

Reply:
@roshan-bajgain , I have made the suggested ammendments. Please check

Reply:
Hi, @kiboma2021 !

I notice that you applied the requested changes and edited your original comment. Great job! :+1: